setup.py

```diff
@@ -280,30 +286,35 @@ def get_extensions():
     libpng = shutil.which("libpng-config")
     pngfix = shutil.which("pngfix")
     png_found = libpng is not None or pngfix is not None
-    print(f"PNG found: {png_found}")
+    if not use_png and png_found:
+        print(f"{colors.WARNING}Found PNG library, but PNG extension is disabled by compile flag{colors.ENDC}")
+    if use_png and not png_found:
+        print(f"{colors.WARNING}Could not find PNG library{colors.ENDC}")
```

Comment:
Nit: The logic here is correct but complex. I would prefer to just put a message on the else of `use_png` check instead.